bakerydemo/static/css/main.css

```diff
@@ -14,81 +14,162 @@
 /* 9. Miscellaneous/ Helper styles ------------------------------------------ */
 
 /* Global styles */
+/* CSS Variables */
+:root {
+  --black: #000000;
+  --dark: #333333;
```

Comment:
question: any particular reason why this var has a descriptive name rather than a colour name like the others?

Reply:
This is the name of the colour in the designs, I'm afraid I just copied these colour names directly as they are what I'll be referencing while developing. Could be awkward for future developers though